app/views/providers/limitations/show.html.erb

```diff
@@ -0,0 +1,35 @@
+<%= page_template(
+      page_title: t('.h1-heading')
+    ) %>
+
+<% if @legal_aid_application.proceeding_types.present? %>
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
```

Comment:
if you use the block version of the template (`<%= page_template(page_title: t('.h1-heading')) do %>`) around everything, you wouldn't need these 2 divs